### PR TITLE
Sign out (React)

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:
Documents how to sign out of a React app.

### Resolves:

* [OKTA-221202](https://oktainc.atlassian.net/browse/OKTA-221202)

reactdocs

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+After performing [local signout](/docs/guides/sign-users-out/react/sign-out-of-your-app/), navigate the user's browser to the [OIDC logout page](https://developer.okta.com/docs/reference/api/oidc/#logout).
+
+This page will clear the user's Okta session then redirect back to the `post_logout_redirect_uri` provided. This URI must be one of those listed in the `Logout redirect URI` section of your application's settings.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md]: This page clears the user's Okta session, and then redirects back to the `post_logout_redirect_uri` that is provided.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Open your Okta Developer Console:
+
+<a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
+
+1. Select **Applications**, and then pick your application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md]: Select **Applications**, and then select your application.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Select **General** and click **Edit**.
+
+3. In the **Logout redirect URI section**, add the **Base URI** of your application, optionally followed by a path, for example, `http://localhost:8080/logged_out`. Also, add any URIs where your application runs in production, such as `https://app.example.com/logged_out`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md]: 3. In the **Logout redirect URI section**, add the **Base URI** of your application. You can optionally follow that URI with a path, for example, `http://localhost:8080/logged_out`.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+Call the `logout` method on the [Auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authlogouturi) object. The [withAuth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#withauth) [higher-order component](https://reactjs.org/docs/higher-order-components.html) makes this object easily available within components as `this.props.auth`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md]: .... makes this object easily available within components such as `this.props.auth`.

Comment (aarongranick-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md]: In this case `this.props.auth` isn't an example, it is literally the variable it is exposed as.

Comment (aarongranick-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md]: How about "easily available within your components". (We continue to show you exactly how to use it)

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md]: > In this case `this.props.auth` isn't an example, it is literally/exactly the variable name we expose.

Okay, so the word "as" is throwing me off....is there another way to say it?

...makes this object easily available within `this.props.auth` components???

**Pull request merged**